config/dataset/single_tile.yaml

```diff
@@ -4,3 +4,5 @@ params:
   n_batches: 10
   batch_size: 6400
   num_workers: 0
+  generate_device: "cuda:0"
```

Comment:
there is a `gpus` argument in the main config file, is there a way you can propagate that value to this one?  (to avoid specifying two devices in two different places)

Reply:
I did this because this is the device for loading data and can be different from the device for training model. We may or may not use Data Parallel, but in that case, this `generate_device` is paired with a cpu process to generate data for models. And we would have multiple models on multiple gpus and those gpus might be different from `generate_device`. I guess I did this because I was trying to avoid the confusion of two different `decoder` objects potentially on two different devices as mentioned in the descriptions I just add.

But I can remove this and use the one from the main config file.